addons/mail/tests/test_mail_channel.py

```diff
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from datetime import datetime
+from unittest.mock import patch
+
+from odoo import Command, fields
```

Comment:
Command is already imported one line below

Reply:
removed